app/src/main/res/values-v31/colors.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <color name="md_theme_primary">@android:color/system_accent1_200</color>
+    <color name="md_theme_onPrimary">@android:color/system_accent1_0</color>
```

Comment:
Where did you get these values from? Some online tool?

Reply:
https://m3.material.io/theme-builder